runtime/vm/jnimisc.cpp

```diff
@@ -411,7 +411,9 @@ getVersion(JNIEnv *env)
 	J9VMThread *currentThread = (J9VMThread*)env;
 	J9JavaVM *vm = currentThread->javaVM;
 
-	if (J2SE_VERSION(vm) >= J2SE_19) {
+	if (J2SE_VERSION(vm) >= J2SE_V10) {
+		return JNI_VERSION_10;
+	} else if (J2SE_VERSION(vm) >= J2SE_19) {
```

Comment:
Should this be == now that there's a new constant for 10?

Reply:
the result should be the same. I will add the == in.

Reply:
> The only concern I have is that the code allows using newer JNI constants with older vms.
@DanHeidinga do you mean for versions below 1.8?